www/src/data/sidebars/doc-links.yaml

```diff
@@ -248,6 +248,8 @@
           items:
             - title: What are Gatsby Themes?
               link: /docs/themes/what-are-gatsby-themes/
+            - title: Getting Started
```

Comment:
I left a note to update the h1 for SEO, but this is fine as-is in the sidebar (and breadcrumb) since it will sit next to "Themes" for context.